svthalia/Reaxit
# Liked photos

Closes #304 .

### Summary
This adds a screen accesible from the albums screen, where people can view their liked photos.

### How to test
Steps to test the changes you made:
1. Go to Albums screen.
2. Click on the heart icon.
3. See your liked photos.
